src/components/DatePicker.js

```diff
@@ -21,14 +21,14 @@ class DatePicker extends React.Component {
       this._handleBlur();
     }
     this.setState({
-      selectedDate: date
+      selectedDate: moment.unix(date)
     });
 
     this.props.onDateSelect(moment.unix(date).format(this.props.format));
```

Comment:
If we send the component a unix timestamp, we should return the date in the same format.